tests/periph_rtc/main.c

```diff
@@ -53,16 +54,7 @@ static void inc_secs(struct tm *time, unsigned val)
 
 static void cb(void *arg)
 {
-    (void)arg;
-
-    puts("Alarm!");
-
-    if (++cnt < REPEAT) {
-        struct tm time;
-        rtc_get_alarm(&time);
-        inc_secs(&time, PERIOD);
-        rtc_set_alarm(&time, cb, NULL);
-    }
+    mutex_unlock((mutex_t *)arg);
```

Comment:
You don't need to cast the `void*`

Reply:
Changed. I squashed it directly, I hope that's ok for this small change.